components/submissions/StatusBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+import { tw } from '../../utils';
+
+interface Props {
```

Comment:
StatusBadgeProps please! Get used to giving proper names to things that are very likely to be exported